### PR TITLE
chore(3id): invitees

### PR DESCRIPTION
**Pull request by maurerbot**

### CODE REVIEW

File: 3iD/src/screens/onboarding/FAQ/Accordion/AccordionHowToUse.tsx [hunk 1]
```diff
@@ -11,9 +11,10 @@ import {
 type HowToProps = {
   account: undefined | null | string;
   defaultExpanded: boolean;
+  collapsable: boolean;
```
Comment (crimeminister) [inline 3iD/src/screens/onboarding/FAQ/Accordion/AccordionHowToUse.tsx:14]:
```suggestion
  collapsible: boolean;
```

File: 3iD/src/screens/onboarding/FAQ/Accordion/AccordionHowToUse.tsx [hunk 2]
```diff
@@ -11,9 +11,10 @@ import {
...
 };
 
-const AccordionHowToUse = ({ account, defaultExpanded }: HowToProps) => {
+const AccordionHowToUse = ({ account, defaultExpanded, collapsable }: HowToProps) => {
```
Comment (crimeminister) [inline 3iD/src/screens/onboarding/FAQ/Accordion/AccordionHowToUse.tsx:17]:
```suggestion
const AccordionHowToUse = ({ account, defaultExpanded, collapsible }: HowToProps) => {
```

File: 3iD/src/screens/onboarding/FAQ/Accordion/AccordionSellInvite.tsx [hunk 3]
```diff
@@ -10,8 +10,10 @@ import {
 
 const AccordionSellInvite = ({
   defaultExpanded,
+  collapsable,
```
Comment (crimeminister) [inline 3iD/src/screens/onboarding/FAQ/Accordion/AccordionSellInvite.tsx:13]:
```suggestion
  collapsible,
```

File: 3iD/src/screens/onboarding/FAQ/Accordion/AccordionSellInvite.tsx [hunk 4]
```diff
@@ -10,8 +10,10 @@ import {
...
 }: {
   defaultExpanded: boolean;
+  collapsable: boolean;
```
Comment (crimeminister) [inline 3iD/src/screens/onboarding/FAQ/Accordion/AccordionSellInvite.tsx:16]:
```suggestion
  collapsible: boolean;
```

File: 3iD/src/screens/onboarding/FAQ/Accordion/AccordionSellInvite.tsx [hunk 5]
```diff
@@ -25,13 +27,15 @@ const AccordionSellInvite = ({
   }
 
   const handleToggle = () => {
+    if (collapsable) {
```
Comment (crimeminister) [inline 3iD/src/screens/onboarding/FAQ/Accordion/AccordionSellInvite.tsx:30]:
```suggestion
    if (collapsible) {
```

File: 3iD/src/screens/onboarding/FAQ/Accordion/AccordionWhatIsPFP.tsx [hunk 6]
```diff
@@ -10,8 +10,10 @@ import {
 
 const AccordionWhatIsPFP = ({
   defaultExpanded,
+  collapsable,
```
Comment (crimeminister) [inline 3iD/src/screens/onboarding/FAQ/Accordion/AccordionWhatIsPFP.tsx:13]:
```suggestion
  collapsible,
```

File: 3iD/src/screens/onboarding/FAQ/Accordion/AccordionWhatIsPFP.tsx [hunk 7]
```diff
@@ -10,8 +10,10 @@ import {
...
 }: {
   defaultExpanded: boolean;
+  collapsable: boolean;
```
Comment (crimeminister) [inline 3iD/src/screens/onboarding/FAQ/Accordion/AccordionWhatIsPFP.tsx:16]:
```suggestion
  collapsible: boolean;
```

File: 3iD/src/screens/onboarding/FAQ/Accordion/AccordionWhatIsPFP.tsx [hunk 8]
```diff
@@ -25,13 +27,15 @@ const AccordionWhatIsPFP = ({
...
 
   const handleToggle = () => {
-    const state = expanded;
-    setExpanded(!expanded);
-    setStyles(
-      state
-        ? { width: 14, height: 7, transform: [{ rotate: "0deg" }] }
-        : { width: 14, height: 7, transform: [{ rotate: "180deg" }] }
-    );
+    if (collapsable) {
```
Comment (crimeminister) [inline 3iD/src/screens/onboarding/FAQ/Accordion/AccordionWhatIsPFP.tsx:30]:
```suggestion
    if (collapsible) {
```

File: 3iD/src/screens/onboarding/FAQ/Accordion/AccordionWhoIsBehind.tsx [hunk 9]
```diff
@@ -10,8 +10,10 @@ import {
 
 const AccordionWhoIsBehind = ({
   defaultExpanded,
+  collapsable,
```
Comment (crimeminister) [inline 3iD/src/screens/onboarding/FAQ/Accordion/AccordionWhoIsBehind.tsx:13]:
```suggestion
  collapsible,
```

File: 3iD/src/screens/onboarding/FAQ/Accordion/AccordionWhoIsBehind.tsx [hunk 10]
```diff
@@ -10,8 +10,10 @@ import {
...
 }: {
   defaultExpanded: boolean;
+  collapsable: boolean;
```
Comment (crimeminister) [inline 3iD/src/screens/onboarding/FAQ/Accordion/AccordionWhoIsBehind.tsx:16]:
```suggestion
  collapsible: boolean;
```

**Review by alfl — APPROVED** [review]
Approved in spite of crazy percentage calculation.

File: 3iD/src/screens/onboarding/FAQ/Accordion/AccordionWhoIsBehind.tsx [hunk 11]
```diff
@@ -25,13 +27,15 @@ const AccordionWhoIsBehind = ({
...
 
   const handleToggle = () => {
-    const state = expanded;
-    setExpanded(!expanded);
-    setStyles(
-      state
-        ? { width: 14, height: 7, transform: [{ rotate: "0deg" }] }
-        : { width: 14, height: 7, transform: [{ rotate: "180deg" }] }
-    );
+    if (collapsable) {
```
Comment (crimeminister) [inline 3iD/src/screens/onboarding/FAQ/Accordion/AccordionWhoIsBehind.tsx:30]:
```suggestion
    if (collapsible) {
```

File: 3iD/src/screens/onboarding/FAQ/index.tsx [hunk 12]
```diff
@@ -39,10 +40,11 @@ const FAQ = ({ account, inviteCode }: Account) => {
         FAQ
       </Text>
 
-      <AccordionHowToUse account={account} defaultExpanded={true} />
-      <AccordionSellInvite defaultExpanded={useBreakpoint(true, false)} />
-      <AccordionWhatIsPFP defaultExpanded={useBreakpoint(true, false)} />
-      <AccordionWhoIsBehind defaultExpanded={useBreakpoint(true, false)} />
+      <AccordionHowToUse account={account} defaultExpanded={true} collapsable={true}/>
```
Comment (crimeminister) [inline 3iD/src/screens/onboarding/FAQ/index.tsx:43]:
```suggestion
      <AccordionHowToUse account={account} defaultExpanded={true} collapsible={true}/>
```

File: 3iD/src/screens/onboarding/FAQ/index.tsx [hunk 13]
```diff
@@ -39,10 +40,11 @@ const FAQ = ({ account, inviteCode }: Account) => {
...
+      <AccordionSellInvite defaultExpanded={false} collapsable={true}/>
```
Comment (crimeminister) [inline 3iD/src/screens/onboarding/FAQ/index.tsx:44]:
```suggestion
      <AccordionSellInvite defaultExpanded={false} collapsible={true}/>
```

File: 3iD/src/screens/onboarding/FAQ/index.tsx [hunk 14]
```diff
@@ -39,10 +40,11 @@ const FAQ = ({ account, inviteCode }: Account) => {
...
+      <AccordionWhatIsPFP defaultExpanded={false} collapsable={true}/>
```
Comment (crimeminister) [inline 3iD/src/screens/onboarding/FAQ/index.tsx:45]:
```suggestion
      <AccordionWhatIsPFP defaultExpanded={false} collapsible={true}/>
```

File: 3iD/src/screens/onboarding/FAQ/index.tsx [hunk 15]
```diff
@@ -39,10 +40,11 @@ const FAQ = ({ account, inviteCode }: Account) => {
...
+      <AccordionWhoIsBehind defaultExpanded={false} collapsable={true}/>
```
Comment (crimeminister) [inline 3iD/src/screens/onboarding/FAQ/index.tsx:46]:
```suggestion
      <AccordionWhoIsBehind defaultExpanded={false} collapsible={true}/>
```

**Review by crimeminister — CHANGES_REQUESTED** [review]

**Pull request merged**